### PR TITLE
Custom Actions

### PR DESCRIPTION
**Pull request by orospakr**
Adds docs for the Custom Actions feature!

Also adds rbenv `.ruby-version` file and updates README to have people use that instead. Safe and more standard approach than asking them to manually install rubies globally.

BTW actually in general I want to avoid PR process for doc updates, at least for now at small team scale. For many trivial doc updates I want us to just go ahead and push commits right onto `master`. Low friction to doc updates is key to devs doing their part to maintain freshness in the docs and dealing with papercuts as they see them.

BUT for this particular change @Moux I'd like you to review it so we can discuss since I changed install procedure.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.7.3
```
Comment (orospakr) [inline .ruby-version:1]: this is the only actual technical change made to support using rbenv

File: README.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+# Developer Portal Setup
 
 The developer portal is built using Jekyll and runs on GitHub Pages.  More information can be found at: <a href="https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll">Testing your GitHub Pages site locally with Jekyll</a>
 
 ## Prerequisites:
-  1. Have Ruby installed.  This is installed by default on MacOS. Instructions for all platforms are here: <a href="https://www.ruby-lang.org/en/documentation/installation/">Installing Ruby</a>
-  2. Have Bundler installed.  Instructions are available here: <a href="https://bundler.io">Installing Bundler</a>
-  3. Have Jekyll installed.  Installation instructions are available here: <a href="https://jekyllrb.com">Installing Jekyll</a>. Please note that Windows isn't officially supported by Jekyll.
-  
+1. Ensure you have rbenv (a tool for managing separate copies of Ruby for different projects) installed and setup (install with Homebrew: `brew install rbenv`). Be sure to set up your PATH and environment afterwards as per instructions given by `rbenv init`.
```
Comment (orospakr) [inline README.md:6]: link to rbenv, qualify homebrew instruction with osx.

File: _android/custom-actions.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+<section id="{{ page.title | slugify }}" markdown=1>
+# Handling Custom Actions
+
+Judo layers support a custom type for actions alongside the Open URL, Present URL, and Close action types. This can be used to implement the behavior for custom buttons in your own code.
+
+The SDK provides an API for you to register a callback to be notified of a custom action being activated (typically by a user's tap).
+
+<p class="note">
+<span class="bold">Note: </span>
```
Comment (orospakr) [inline _android/custom-actions.md:17]: try using markdown=1 with *s for bold instead

**Comment by Moux** [comment]
I ended up having to run `sudo gem install jekyll bundler` and `sudo gem install github-pages bundler` in order to force it to install those gems and dependencies on my current ruby version.  Other than that, it worked.

**Comment by orospakr** [comment]
you shouldn't need to use sudo @Moux . perhaps your rbenv wasn't set up correctly

**Review by Moux — APPROVED** [review]
Looks like I was wrong about the html tags inside markdown=1

**Pull request merged**